packages/graph/hash_graph/tests/integration/postgres/entity.rs

```diff
@@ -90,7 +94,7 @@ async fn update() {
         .await
         .expect("could not seed database:");
 
-    let identifier = api
+    let metadata = api
```

Comment:
(comment location not accurate)
Something that came up while chatting was to have tests for testing how the metadata persists for updates. 
Would be good to have testing to see if 2 users updating an entity behaves as expected in terms of the returned metadata.

This would be a follow-up, though. Just making sure we're capturing the intention to have something like that :)

Reply:
> Would be good to have testing to see if 2 users updating an entity behaves as expected in terms of the returned metadata.

Agreed, but this requires the `updated_by_id` field introduced in [this follow-up PR](https://github.com/hashintel/hash/pull/1197), right? Or am I misunderstanding.

If not, I will add this test as part of that PR.